src/map/entities/battleentity.cpp

```diff
@@ -1419,7 +1419,7 @@ void CBattleEntity::OnCastFinished(CMagicState& state, action_t& action)
         // TODO: actual radius calculation
         float radius = spell::GetSpellRadius(PSpell, this);
 
-        PAI->TargetFind->findWithinCone(PActionTarget, radius, 45, flags);
+        PAI->TargetFind->findWithinCone(PActionTarget, radius, 90, flags);
```

Comment:
Last thing I guess.... What is this change?

Reply:
This is used in player magic conal spells. the only such ones I could find were all BLU spells but a 45° cone is a bit small from what I remember on retail so I increased it to 90°. This gives Breath spells some room to have some noticeable damage modulation when the monster is not as much in front, as it was on retail. This pairs with Breath stuff in bluemagic.lua:
![image](https://user-images.githubusercontent.com/16270541/208326170-c94dac25-19b7-4a29-8fb7-fefdd16dc066.png)
Of course this also works with a smaller cone it's just less visible.

Reply:
that would be a 45 / 256 irrc. Not a 45 / 360

Reply:
ok, I'll put it back to 45 and restrict the angle in Breath stuff in bluemagic.lua as well